smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/GetTogglesIT.java

```diff
@@ -57,13 +58,28 @@ public static void beforeClass() {
      * Verifies if the "ENABLED" flag is always enabled on Author Instance
      * Considers the test failed if the answer can't be parsed.
      *
-     * @throws ClientException if an error occurred
+     * @throws InterruptedException if an error occurred
      */
     @Test
-    public void testTogglesEndpointReturnsStaticEnabledFlagInJsonResponseOnAuthor() throws ClientException {
+    public void testTogglesEndpointReturnsStaticEnabledFlagInJsonResponseOnAuthor() throws InterruptedException {
         ObjectMapper mapper = new ObjectMapper();
-        SlingHttpResponse response = adminAuthor.doGet("etc.clientlibs/toggles.json", 200);
```

Comment:
@bogdananton , thanks for the PR. 

AFAIU you don't need to retry in your test. The `ServerErrorRetryStrategy` in the Sling testing clients [1] should take care of that. By default, it retries on `5XX` response codes. You can customize the retry with various system properties [2].

The retries are not visible by default, you can turn the logs on using [3]. 

Does this help for your retry needs, or do you see something missing for your use case?

[1] https://github.com/apache/sling-org-apache-sling-testing-clients/blob/master/src/main/java/org/apache/sling/testing/clients/util/ServerErrorRetryStrategy.java

[2]https://github.com/apache/sling-org-apache-sling-testing-clients/blob/master/src/main/java/org/apache/sling/testing/clients/SystemPropertiesConfig.java

[3] https://github.com/apache/sling-org-apache-sling-testing-clients/blob/master/src/main/java/org/apache/sling/testing/clients/SystemPropertiesConfig.java#L56

Reply:
Yes, that helps, I'm closing the PR. Thank you!